source/strings.tex

```diff
@@ -1189,7 +1189,7 @@
 
 \pnum
 \effects
-Equivalent to \tcode{traits::copy(s, data() + pos, rlen)}.
+Equivalent to: \tcode{traits::copy(s, data() + pos, rlen)}.
```

Comment:
The current form is in line with https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#writing-effects-in-a-function-description